### PR TITLE
remove unnecessary space

### PR DESCRIPTION
**Pull request by CoalSephos**
L16, L26, L44, L63, L73, L83 space after second set of ** removed

### CODE REVIEW

File: src/modules/block.js [hunk 1]
```diff
@@ -13,7 +13,7 @@ module.exports = bot => {
       msg.channel.createMessage(`Blocked <@${userId}> (id ${userId}) from modmail`);
     }
 
-    let logText = `**Blocked: ** ${thread.user_name} (${thread.user_id}) was blocked.`;
+    let logText = `**Blocked: **${thread.user_name} (${thread.user_id}) was blocked.`;
```
Comment (bsian03) [inline src/modules/block.js:16]:
```suggestion
    let logText = `**Blocked:** ${thread.user_name} (${thread.user_id}) was blocked.`;
```
Just so it looks neater

File: src/modules/block.js [hunk 2]
```diff
@@ -23,7 +23,7 @@ module.exports = bot => {
       const reason = args.slice(1).join(' ').trim();
 
       if (reason && reason.length) {
-        logText = `**Blocked: ** ${thread.user_name} (${thread.user_id}) was blocked for ${reason}`;
+        logText = `**Blocked: **${thread.user_name} (${thread.user_id}) was blocked for ${reason}`;
```
Comment (bsian03) [inline src/modules/block.js:26]:
```suggestion
        logText = `**Blocked:** ${thread.user_name} (${thread.user_id}) was blocked for ${reason}`;
```

File: src/modules/block.js [hunk 3]
```diff
@@ -41,7 +41,7 @@ module.exports = bot => {
 
       if (reason && reason.length) {
         text = `You have been blocked for ${reason}`;
-        logText = `**Blocked: ** ${thread.user_name} (${thread.user_id}) was blocked for ${reason}`;
+        logText = `**Blocked: **${thread.user_name} (${thread.user_id}) was blocked for ${reason}`;
```
Comment (bsian03) [inline src/modules/block.js:44]:
```suggestion
        logText = `**Blocked:** ${thread.user_name} (${thread.user_id}) was blocked for ${reason}`;
```

File: src/modules/block.js [hunk 4]
```diff
@@ -60,7 +60,7 @@ module.exports = bot => {
       msg.channel.createMessage(`Unblocked <@${userId}> (id ${userId}) from modmail`);
     }
 
-    let logText = `**Unblocked: ** ${thread.user_name} (${thread.user_id}) was unblocked.`;
+    let logText = `**Unblocked: **${thread.user_name} (${thread.user_id}) was unblocked.`;
```
Comment (bsian03) [inline src/modules/block.js:63]:
```suggestion
    let logText = `**Unblocked:** ${thread.user_name} (${thread.user_id}) was unblocked.`;
```

File: src/modules/block.js [hunk 5]
```diff
@@ -70,7 +70,7 @@ module.exports = bot => {
       const reason = args.slice(1).join(' ').trim();
 
       if (reason && reason.length) {
-        logText = `**Unblocked: ** ${thread.user_name} (${thread.user_id}) was unblocked for ${reason}`;
+        logText = `**Unblocked: **${thread.user_name} (${thread.user_id}) was unblocked for ${reason}`;
```
Comment (bsian03) [inline src/modules/block.js:73]:
```suggestion
        logText = `**Unblocked:** ${thread.user_name} (${thread.user_id}) was unblocked for ${reason}`;
```

File: src/modules/block.js [hunk 6]
```diff
@@ -80,7 +80,7 @@ module.exports = bot => {
       const reason = args.join(' ').trim();
 
       if (reason && reason.length) {
-        logText = `**Unblocked: ** ${thread.user_name} (${thread.user_id}) was unblocked for ${reason}`;
+        logText = `**Unblocked: **${thread.user_name} (${thread.user_id}) was unblocked for ${reason}`;
```
Comment (bsian03) [inline src/modules/block.js:83]:
```suggestion
        logText = `**Unblocked:** ${thread.user_name} (${thread.user_id}) was unblocked for ${reason}`;
```

**Comment by CoalSephos** [comment]
updates based upon reviews

**Pull request merged**